src/Configuration/HiveActionConfigRowDefinition.php

```diff
@@ -4,11 +4,11 @@
 
 namespace Keboola\DbExtractor\Configuration;
 
-use Keboola\DbExtractorConfig\Configuration\ActionConfigRowDefinition;
+use Keboola\DbExtractorConfig\Configuration\GetTablesListFilterDefinition;
 use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
 use Symfony\Component\Config\Definition\Builder\TreeBuilder;
 
-class HiveActionConfigRowDefinition extends ActionConfigRowDefinition
+class HiveActionConfigRowDefinition extends GetTablesListFilterDefinition
```

Comment:
Všimol som si, ze na CSAS produkcii timeoutuje `getTables` akcia, preto som to upravil aby bolo mozne nacitat tabulky a stlpce osobitne.

Jedine co bolo potrebne upravit je tato konfiguracia, ostatne triedy to uz podporuju.
+ dopisal som na to testy.